src/main/java/org/inferred/freebuilder/processor/property/Property.java

```diff
@@ -38,6 +38,12 @@ public FieldAccess getField() {
   /** Returns true if getters start with "get"; setters should follow suit with "set". */
   public abstract boolean isUsingBeanConvention();
 
+  /** Returns true if the field is to appear in the toString. */
+  public abstract boolean isInToString();
```

Comment:
I think it would be reasonable to default these to true, since there are only two tests that set them otherwise.

Reply:
Yeah that sounds reasonable, I was thinking the same thing. I was having trouble getting the build to generate the Property implementations for me. Does Property_Builder get generated as part of `./gradlew test`, or is there another step ? 
Thanks.

Reply:
See Constructor below for defaulting behavior.

Reply:
The _Builders are currently updated by adding the processor in the IDE and having it overwrite them. I would prefer something more automated, but it's kind of chicken-and-egg — you need the processor before you can build the processor.